packages/api-derive/src/staking/stakerRewards.ts

```diff
@@ -48,8 +48,8 @@ function parseRewards (api: DeriveApi, stashId: AccountId, [erasPoints, erasPref
         const valCut = valComm.mul(avail).div(BN_BILLION);
         let staked: BN;
 
-        if (validatorId === stakerId) {
-          staked = exposure.own.unwrap();
+        if (validatorId === stakerId && (exposure as PalletStakingExposure).own) {
```

Comment:
Not sure if this is valid. Need to double check

Reply:
I need to add the storage call to `ErasStakersOverview `, and do this correctly so we don't need to come back to it.